### ISSUE TITLE
ui5-combobox aria-labelledby not working when inside ui5-busyindicator 

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#2903]
**Describe the bug**
When a ui5-combobox is wrapped inside a ui5-busyindicator, the aria-labelledby attribute is not working properly.

**To reproduce**
1. Go to https://codesandbox.io/s/ui5-webcomponents-forked-8hdz3
2. using Speech Reader like NVDA, tab into the ui5-combobox
3. notice that the ui5-label is not read

**Expected behavior**
The label should be read out "LABEL combo box collapsed ..." instead of "PLACEHOLDER combo box collapsed", the same way it does when the combobox is not wrapped in a ui5-busyindicator

**Screenshots**
combobox:
![image](https://user-images.githubusercontent.com/24360164/109890367-05ac6c00-7c3c-11eb-811e-f3cf0789ee4e.png)
screen reader output:
![image](https://user-images.githubusercontent.com/24360164/109890378-0b09b680-7c3c-11eb-9a4a-c25d3b5d1d19.png)

**Context**
- UI5 Web Components version 0.29.4

**Affected components** 
ui5-combobox
ui5-busyindicator

**Organization:** 
SuccessFactors

**Priority:** 
Medium

Thank you for looking into this :)

**Comment by fifoosid** [comment]
Is this issue NVDA only related, because I was unable to reproduce it with VoiceOver? See screenshot:

![image](https://user-images.githubusercontent.com/16850339/109915307-4054e980-7cba-11eb-8498-f1e40ac1e747.png)

**Comment by ilhan007** [comment]
Closing due to inactivity.
